pkg/odo/cli/component/devfile.go

```diff
@@ -124,7 +117,7 @@ func (do *DeployOptions) DevfileBuild() (err error) {
 	// Apply ignore information
 	err = genericclioptions.ApplyIgnore(&do.ignores, do.sourcePath)
 	if err != nil {
-		return errors.Wrap(err, "unable to apply ignore information")
+		return errors.Wrap(err, "Unable to apply ignore information")
```

Comment:
Should be lowercase 'u' at the start to match the other errors.Wrap statements throughout the codebase!